src/_companies/north-america/united-states/dc/commonlit/index.md

```diff
@@ -0,0 +1,27 @@
+---
+name:  "CommonLit"
+
+continent: "North America"
+country: "United States"
+region: "DC"
```

Comment:
I must admit I'm not well educated on this, but isn't the federal district also called "Washington DC"? I think it might be better to use it as a region name and also as part of the folder name (`washington-dc`), since that's used as a slug for the page.

Getting this right is important, since once there will be a critical mass of companies the idea is to split them into sub- categories/pages. I think it would look weird to only have "DC" as the state within the US then (I might be totally wrong though).

@geoffharcourt, what do you think? :)